Detectors/TPC/base/src/ParameterGEM.cxx

```diff
@@ -12,19 +12,47 @@
 /// \brief Implementation of the parameter class for the GEM stack
 /// \author Andi Mathis, TU München, andreas.mathis@ph.tum.de
 
+// Remark: This file has been modified by Viktor Ratza in order to
+// implement the efficiency models for the collection and the
+// extraction efficiency.
+
 #include "TPCBase/ParameterGEM.h"
 
 using namespace o2::TPC;
 
 ParameterGEM::ParameterGEM()
-  : mAbsoluteGain(),
+  : mGeometry(),
+    mPotential(),
+    mElectricField(),
+    mAbsoluteGain(),
     mCollectionEfficiency(),
     mExtractionEfficiency()
 {}
 
-void ParameterGEM::setDefaultValues()
+void ParameterGEM::setDefaultValues() 
 {
+
+  mGeometry = {{1, 3, 3, 1}};
+  mPotential = {{270.f, 250.f, 270.f, 340.f}};
+  mElectricField = {{0.4f, 4.f, 2.f, 0.1f, 4.f}};
   mAbsoluteGain = {{14.f, 8.f, 53.f, 240.f}};
-  mCollectionEfficiency = {{1.f, 0.2f, 0.25f, 1.f}};
-  mExtractionEfficiency = {{0.65f, 0.55f, 0.12f, 0.6f}};
+  //mCollectionEfficiency = {{1.f, 0.2f, 0.25f, 1.f}};
+  //mExtractionEfficiency = {{0.65f, 0.55f, 0.12f, 0.6f}};
+  
+  //Load efficiencies for stack configuration
+  EffGEM = new EfficiencyGEM();
```

Comment:
Since EfficiencyGEM seems to be only used here, why don't you simple use:
`EfficiencyGEM effGEM;`
here and remove the then unneeded member variable `EffGEM`. This would make the code much cleaner.